### PR TITLE
small metadata optimizations

### PR DESCRIPTION
**Pull request by nathanosdev**

### CODE REVIEW

File: src/frontend/assets/hash.svg
Comment (taggrx) [inline src/frontend/assets/hash.svg:1]: What has changed here actually?

File: src/frontend/assets/logo.min.svg
Comment (taggrx) [inline src/frontend/assets/logo.min.svg:1]: And here

File: src/backend/assets.rs [hunk 1]
```diff
@@ -70,6 +70,15 @@ pub fn load() {
         include_bytes!("../../dist/frontend/apple-touch-icon.png").to_vec(),
     );
 
+    add_asset(
```
Comment (taggrx) [inline src/backend/assets.rs:73]: I don't understand this part. Why is it needed? Is it loaded somewhere at all?

File: webpack.config.js [hunk 2]
```diff
@@ -76,6 +75,16 @@ module.exports = {
         new HtmlWebpackPlugin({
             template: path.join(__dirname, asset_entry),
             cache: false,
+            minify: isDevelopment ? false : {
```
Comment (taggrx) [inline webpack.config.js:78]: Nice! Any noticeable improvements in frontend code size?

File: src/backend/assets.rs (hunk 1 above)
Comment (nathanosdev) [reply src/backend/assets.rs:73]: It's set as the image for socials in the `index.html`: https://github.com/TaggrNetwork/taggr/blob/main/src/frontend/src/index.html#L8-L10

So when socials try to load that image it doesn't work and it looks like this:
image

File: src/frontend/assets/hash.svg
Comment (nathanosdev) [reply src/frontend/assets/hash.svg:1]: I had tried to optimize these with https://jakearchibald.github.io/svgomg/ but it caused issues so I decided to defer to a later date and then when I reverted the changes I deleted newlines from the original file by accident. I'll revert properly now.

File: webpack.config.js (hunk 2 above)
Comment (nathanosdev) [reply webpack.config.js:78]: Nothing worth writing home about 😆 

2.54kb -> 2.25kb

**Review by chmllr — APPROVED** [review]

**Review by taggrx — APPROVED** [review]

**Pull request merged**